packages/connect-web/src/impl/core-in-iframe.ts

```diff
@@ -193,19 +204,13 @@ export class CoreInIframe implements ConnectFactoryDependencies {
             if (!this._popupManager) {
                 this._popupManager = this.initPopupManager();
             }
-            this._popupManager.request();
 
             // auto init with default settings
             try {
                 await this.init(this._settings);
             } catch (error) {
                 if (this._popupManager) {
-                    // Catch fatal iframe errors (not loading)
```

Comment:
cool, so this means that these errors are no longer fatal and we can retry usign a different way